OpenEdXMobile/src/main/java/org/edx/mobile/model/course/BlockType.java

```diff
@@ -23,6 +23,7 @@
     VIDEO{ @Override public boolean isContainer() {return false;} },
     HTML{ @Override public boolean isContainer() {return false;} },
     PROBLEM{ @Override public boolean isContainer() {return false;} },
+    OPENASSESSMENT{ @Override public boolean isContainer() {return false;} },
```

Comment:
is there a specific reason this was added below PROBLEM type?

Usually, its a good practice to add all new stuff at the end unless a specific pattern like alphabetic sorting is being followed (which is not in this case).

Reply:
updated, moved it to end